package.json

```diff
@@ -34,42 +34,17 @@
     "postinstall": "lerna bootstrap"
   },
   "dependencies": {
-    "amd-loader": "0.0.5",
-    "babel-cli": "^6.7.5",
-    "babel-core": "^6.17.0",
-    "babel-eslint": "^7.1.0",
-    "babel-loader": "^6.2.4",
-    "babel-plugin-module-resolver": "^2.2.0",
-    "babel-plugin-transform-async-to-generator": "^6.16.0",
-    "babel-plugin-transform-es2015-block-scoping": "^6.7.1",
-    "babel-plugin-transform-es2015-destructuring": "^6.6.5",
-    "babel-plugin-transform-es2015-parameters": "^6.7.0",
-    "babel-plugin-transform-es2015-spread": "^6.6.5",
-    "babel-plugin-transform-flow-strip-types": "^6.14.0",
-    "babel-plugin-transform-runtime": "^6.7.5",
-    "babel-plugin-webpack-alias": "^2.1.1",
-    "babel-polyfill": "^6.7.4",
-    "babel-register": "^6.18.0",
-    "body-parser": "^1.15.0",
-    "check-node-version": "^1.1.2",
     "classnames": "^2.2.5",
     "co": "=4.6.0",
     "codemirror": "^5.1.0",
-    "css-loader": "^0.26.0",
     "devtools-config": "^0.0.9",
     "devtools-local-toolbox": "^0.0.10",
     "devtools-modules": "^0.0.9",
     "devtools-network-request": "^0.0.9",
     "devtools-sham-modules": "^0.0.9",
     "documentation": "^4.0.0-beta11",
     "eslint": "^3.10.0",
-    "eslint-plugin-babel": "^4.0.0",
-    "eslint-plugin-flowtype": "^2.20.0",
-    "eslint-plugin-mozilla": "0.2.3",
-    "eslint-plugin-react": "^6.7.1",
```

Comment:
.eslintrc still refers to rules from all those packages, are you sure we can remove them here ? Getting an error when running npm run lint after a clean yarn install.